packages/plugin-self-paced-reading/docs/jspsych-self-paced-reading.md

```diff
@@ -37,13 +37,12 @@ In addition to the [parameters available in all plugins](https://www.jspsych.org
 
 In addition to the [default data collected by all plugins](https://www.jspsych.org/overview/plugins#data-collected-by-all-plugins), this plugin collects the following data for each trial.
 
-| Name        | Type        | Value                                    |
-| --------    | ----------- | ---------------------------------------- |
-| rt_sentence | number      | Running total time (ms)                  |
-| rt_word     | number      | Individual word time (ms)                |
-| word        | string      | Individual word                          |
-| word_number | number      | Individual word number within sentence   |
-| sentence    | string      | Sentence item                            |
+| Name      | Type     | Value                                                      |
+| --------- | -------- | ---------------------------------------------------------- |
+| words     | string[] | Array of words in the sentence                             |
+| rts       | number[] | Array of reading times for each word (ms)                  |
```

Comment:
I think it would make sense to call this `rt` instead of `rts` to be consistent with other plugins that report multiple response times in an array.

Reply:
Makes sense. I singularized the other plural data properties for consistency.